core/pta/stats.c

```diff
@@ -51,6 +53,16 @@
 
 #define STATS_NB_POOLS			4
 
+/*
+ * STATS_CMD_PRINT_DRIVERS_INFO - Print device drivers information to console
+ *
+ * [in]    value[0].a        Target driver, one of STATS_DRIVER_TYPE_*
+ */
+#define STATS_CMD_PRINT_DRIVERS_INFO	5
```

Comment:
`_DRIVER_` (singular)? since the command prints for one driver

Reply:
ok, makes sense. I'll address that.